components/brave_wallet_ui/page/screens/swap/components/swap/header/header.tsx

```diff
@@ -114,23 +125,27 @@ export const Header = () => {
 
   return (
     <Wrapper>
-      <Row rowHeight='full' verticalAlign='center'>
-        <BraveLogo />
-        <HiddenResponsiveRow maxWidth={570}>
-          <HorizontalDivider
-            height={22}
-            marginRight={12}
-            dividerTheme='darker'
-          />
-          <Text
-            textSize='18px'
-            textColor='text02'
-            isBold={true}
-          >
-            {getLocale('braveSwap')}
-          </Text>
-        </HiddenResponsiveRow>
-      </Row>
+      {!isPanel ? (
+        <Row rowHeight='full' verticalAlign='center'>
+          <BraveLogo />
+          <HiddenResponsiveRow maxWidth={570}>
+            <HorizontalDivider
+              height={22}
+              marginRight={12}
+              dividerTheme='darker'
+            />
+            <Text
+              textSize='18px'
+              textColor='text02'
+              isBold={true}
+            >
+              {getLocale('braveSwap')}
+            </Text>
+          </HiddenResponsiveRow>
+        </Row>
+      ) : (
+        <HorizontalSpacer size={2} />
+      )}
```

Comment:
How about flipping this to check `isPanel` instead of `!isPanel`? (one less operation)

Reply:
Done!